dist/platforms/mac/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+#
+# Run steps
+#
+source $ACTION_FOLDER/platforms/mac/steps/setup.sh
```

Comment:
Perhaps this setup can be done from the TypeScript part instead so we can completely reuse the existing entrypoint and related scripts from the linux variant.

Reasons:
- These files have been empirically improved upon for 3 years, diverging them into mutliple files with 95%  the same content would make our implementation less cohesive.
- These files are intended to be docker runtime, not build time - if anything we should be adding a layer in between that installs it, or mount the installation from the runners host.